wifiphisher/common/interfaces.py

```diff
@@ -662,6 +773,9 @@ def interface_property_detector(network_adapter):
     if pyw.iswireless(network_adapter.name):
         network_adapter.is_wireless = True
```

Comment:
Can you remove this as it is no longer needed and can you also remove the whole method in `NetworkAdapter` up there. Also can you remove the tests as well 😄 .